app/hooks/projects/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface UseSaveProjectProps {
+  requestOptions: {}
```

Comment:
I think we should probably create an interface for the request options as well. The good thing about doing that is that we could reuse that definition in many other hooks.

Reply:
You are right, I'm going to use the AxiosRequestConfig
```
import { AxiosRequestConfig } from 'axios';

export interface UseSaveProjectProps {
  requestConfig: AxiosRequestConfig
}
```